src/components/admin/event/AdminPickupEvent/style.module.scss

```diff
@@ -32,6 +32,12 @@
 .submitButtons {
   display: flex;
   gap: 1rem;
+  height: auto;
+
+  .expandButton {
+    height: auto;
+    min-height: 2.5rem;
+  }
```

Comment:
I think we should enable this for every button (i.e. move this to `Button` so it applies to all buttons) since there's no reason to prevent them from expanding